src/cgroups/v1/cpuacct.rs

```diff
@@ -0,0 +1,57 @@
+use std::{fs, path::Path};
+
+use anyhow::Result;
+use nix::unistd::Pid;
+use oci_spec::LinuxResources;
+
+use crate::cgroups::common::{self, CGROUP_PROCS};
+
+use super::Controller;
+
+const CGROUP_CPUACCT_TASKS: &str = "tasks";
+
+pub struct CpuAcct {}
+
+impl Controller for CpuAcct {
+    fn apply(_linux_resources: &LinuxResources, cgroup_path: &Path, pid: Pid) -> Result<()> {
+        log::debug!("Apply cpuacct cgroup config");
+        fs::create_dir_all(cgroup_path)?;
+
+        Self::apply(cgroup_path, pid)?;
+
+        common::write_cgroup_file(cgroup_path.join(CGROUP_PROCS), pid)?;
+        Ok(())
+    }
+}
+
+impl CpuAcct {
+    fn apply(root_path: &Path, pid: Pid) -> Result<()> {
+        common::write_cgroup_file_str(root_path.join(CGROUP_CPUACCT_TASKS), &pid.to_string())?;
```

Comment:
Why is this required? My understanding is that writing the pid to the cgroup.procs file also moves all of its threads.

Reply:
Thank you for your comment.
I tried it in my environment but this process was unnecessary.
I will commit the fix later.